src/components/LineChart/LineChart.js

```diff
@@ -114,100 +115,105 @@ function LineChart({
+                <g key={`line-plot-${line.id || lineIndex}`}>
+                  <path
+                    d={`
                     M
                     ${getX(0)},
-                    ${getY(line.values[0], progress, chartHeight)}
+                    ${getY(line.values[0], val, chartHeight)}
 
                     ${line.values
                       .slice(1)
                       .map(
                         (val, index) =>
                           `L
-                           ${getX((index + 1) * progress)},
-                           ${getY(val, progress, chartHeight)}
+                           ${getX((index + 1) * val)},
+                           ${getY(val, val, chartHeight)}
                           `
                       )
                       .join('')}
                   `}
-                  fill="transparent"
-                  stroke={line.color || color(lineIndex, { lines })}
-                  strokeWidth="2"
-                />
-                {line.values.slice(1, -1).map((val, index) => (
-                  <circle
-                    key={index}
-                    cx={getX(index + 1) * progress}
-                    cy={getY(val, progress, chartHeight)}
-                    r={dotRadius}
-                    fill="white"
+                    fill="transparent"
                     stroke={line.color || color(lineIndex, { lines })}
-                    strokeWidth="1"
+                    strokeWidth="2"
                   />
-                ))}
-              </g>
-            ))}
-            <line
-              x1={getX(valuesCount - 1) * progress}
-              y1="0"
-              x2={getX(valuesCount - 1) * progress}
-              y2={chartHeight}
-              stroke="#DAEAEF"
-              strokeWidth="3"
-            />
-          </g>
-          {labels && (
-            <g transform={`translate(0,${chartHeight})`}>
-              {labels.map((label, index) => (
-                <text
-                  key={index}
-                  x={getX(index)}
-                  y={LABELS_HEIGHT / 2}
-                  textAnchor={getLabelPosition(index, labels.length)}
-                  fill={labelColor}
-                  css={`
-                    alignment-baseline: middle;
-                    font-size: 12px;
-                    font-weight: 300;
-                    ${unselectable};
-                  `}
-                >
-                  {label}
-                </text>
+                  {line.values.slice(1, -1).map((val, index) => (
+                    <circle
+                      key={index}
+                      cx={getX(index + 1) * val}
+                      cy={getY(val, val, chartHeight)}
+                      r={dotRadius}
+                      fill="white"
+                      stroke={line.color || color(lineIndex, { lines })}
+                      strokeWidth="1"
+                    />
+                  ))}
+                </g>
```

Comment:
Its mean circle now was added on the LineChart component?

Reply:
the `LineChart` i committed by mistake. This is the component where I am having the problems. I was not supposed to commit that. I will revert the changes in the LineChart soon